Maximilian_express_practice/app.js

```diff
@@ -0,0 +1,34 @@
+const express = require("express");
+const bodyParser = require("body-parser");
+const path = require("path");
+
+const app = express();
+const port = 5000;
```

Comment:
why 5000? generally we use port 3000 and only from the env file not directly hard coding it in the code.

Reply:
updated